Orange/widgets/unsupervised/tests/test_owsom.py

```diff
@@ -115,12 +115,31 @@ def test_missing_one_row_data(self):
             self.iris.X[5, 0] = np.nan
 
         self.send_signal(widget.Inputs.data, self.iris)
-        self.assertFalse(widget.Error.no_defined_rows.is_shown())
+        self.assertFalse(widget.Error.not_enough_data.is_shown())
         self.assertTrue(widget.Warning.missing_values.is_shown())
 
         self.send_signal(widget.Inputs.data, None)
         self.assertFalse(widget.Warning.missing_values.is_shown())
 
+    def test_single_row_data(self):
```

Comment:
Could you add `@_patch_recompute_som` before this function, so it won't waste time by actually running SOM?

The time saved is negligible, but it's trivial to do (because the function is already there) and other tests in this module are decorated like this, so let's keep consistency.